gui/canvaspainter/v7/src/TCanvasPainter.cxx

```diff
@@ -99,7 +106,9 @@ class TCanvasPainter: public THttpWSHandler,
 
    virtual void AddDisplayItem(ROOT::Experimental::TDisplayItem *item) final;
 
+   virtual void AddMenuItem(const std::string &name, const std::string &title, const std::string &exec) final;
```

Comment:
Can you inline the function, if you decide to make it `void AddMenuItem(const TMenuItem& menuItem)`? It's too time in that case :-)